docs/variants/msi_z690/faq.md

```diff
@@ -34,29 +34,28 @@ procedure](../recovery).
 
```

Comment:
jeśli chodzi o samodzielne flashowanie to odpowiedź jest OK = nie dajemy gwarancji. Natomiast myślę, o dodaniu zdania, że w przypadku obaw zapraszamy do zakupu sprzętu w naszym sklepie WWW z usługą subskrypcji, w ramach której to my wykonujemy instalację DASHARO i to my bierzemy na siebie ryzyko ujęte w pytaniu "Can I destroy my hardware by installing this firmware?"

Reply:
@rafkoch I have added this information in the newest commit.